File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--programmatically-automatically-fill-ranges-with-incrementally-changing-data_1.cs
        private void AutoFill()
        {
            Excel.Range rng = this.Application.get_Range("B1");
            rng.AutoFill(this.Application.get_Range("B1","B5"), 
                Excel.XlAutoFillType.xlFillWeekdays); 

            rng = this.Application.get_Range("C1"); 
            rng.AutoFill(this.Application.get_Range("C1","C5"),
                Excel.XlAutoFillType.xlFillMonths);

            rng = this.Application.get_Range("D1","D2");
            rng.AutoFill(this.Application.get_Range("D1","D5"),
                Excel.XlAutoFillType.xlFillSeries); 
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--programmatically-automatically-fill-ranges-with-incrementally-changing-data_1.vb
    Private Sub AutoFill()
        Dim rng As Excel.Range = Me.Application.Range("B1")
        rng.AutoFill(Me.Application.Range("B1:B5"), Excel.XlAutoFillType.xlFillWeekdays)

        rng = Me.Application.Range("C1")
        rng.AutoFill(Me.Application.Range("C1:C5"), Excel.XlAutoFillType.xlFillMonths)

        rng = Me.Application.Range("D1:D2")
        rng.AutoFill(Me.Application.Range("D1:D5"), Excel.XlAutoFillType.xlFillSeries)
    End Sub